app/services/leads/search_leads_service.rb

```diff
@@ -0,0 +1,39 @@
+module Leads
+  class SearchLeadsService 
+    include ActiveModel::Model
+
+    attr_reader :results
+    
+    def initialize(request_params, auth_header, jwt)
+      @request_params = request_params
+      @auth_header = auth_header
+      @jwt = jwt
+    end
+
+    def run!
+      leads = if @jwt['params']['is_admin']
+                Lead.where_first_name(@request_params.first_name).where_last_name(@request_params.last_name).where_phone(@request_params.phone).where_email(@request_params.email).where_owner_id(@request_params.owner_id)
+              else
+                Lead.where_first_name(@request_params.first_name).where_last_name(@request_params.last_name).where_phone(@request_params.phone).where_email(@request_params.email).where_owner_id(@request_params.owner_id).where_by_subordinate_users(get_subordinate_user_ids)
```

Comment:
@quoctuan1612 
Chỗ này hãy cho điều kiện where_by_subordinate_users lên trên đầu tiên nhé. (comment break like tương tự comment phía trên nhé)

Reply:
@nistelrooy1984 
Fixed and Committed